packages/system/immucore/collection.yaml

```diff
@@ -21,3 +21,15 @@ packages:
       - https://github.com/kairos-io/immucore
     license: "Apache License v2"
     description: "The Kairos immutability management interface"
+  - name: "immucore"
+    category: "fips-static"
+    ldflags: "-linkmode external -extldflags -static"
```

Comment:
missing ldflags for the non-static fips version

Reply:
no need, using `with`